Util/Xamarin.AndroidBinderator/Xamarin.AndroidBinderator/Engine.cs

```diff
@@ -241,11 +241,13 @@ static async Task DownloadArtifacts(MavenRepository maven, BindingConfig config,
 					}
 				}
 
-
 				if (config.DownloadJavaDocJars)
 				{
 					try
 					{
+						// using (var astrm = await maven.OpenArtifactDocsFile(mavenArtifact.GroupId, mavenArtifact.ArtifactId, version))
```

Comment:
Is this intentionally commented out?

Reply:
yes. Not sure why Redth could not publish MavenNet. He bumped everything after my PR, but nuget is not published yet.
It is not critical, I'll add it in next round. Soon